Dockerfile

```diff
@@ -1,4 +1,8 @@
-FROM fedora
+# Normally it's illadvised to use a FQDN, but this image is auto-built
```

Comment:
Personally curious: Is it? We typically warn against short names (acknowledging that they are convenient.)

Reply:
It's what I've been told/read as a best-practice.  It's more portable to use the short-name in the `Dockerfile` and rely on the build-host to configure the appropriate registry/registries within it's context.  However in practice, esp. with these CI-related Dockerfiles and repository auto-builders (like quay and docker hub), it's often easier to just jam the FQIN in and forget about it.